### PR TITLE
Fix OCP-23030

### PR DESCRIPTION
**Pull request by wewang58**
In some versions, there's no some resources, so fix it. here's [pass log](http://pastebin.test.redhat.com/866680)
@openshift/devexp-qe please check it, thanks

### CODE REVIEW

File: features/registry/registry.feature [hunk 1]
```diff
@@ -80,23 +80,22 @@ Feature: Testing registry
...
+      | (resource: serviceaccounts)?                  |
+      | (name: image-registry-certificates)?          |
+      | (resource: configmaps)?                       |
+      | (name: image-registry-private-configuration)? |
+      | (resource: secrets)?                          | 
+      | (name: image-registry)?                       |
+      | (resource: services)?                         |
+      | (name: node-ca)?                              |
+      | (resource: daemonsets)?                       |
+      | (resource: deployments)?                      |
```
Comment (liangxia) [inline features/registry/registry.feature:98]: No matter those resources exist or not, the scenario will pass when using like above. Then what's the meaning of leaving them there ?

Comment (wewang58) [reply features/registry/registry.feature:98]: some versions it's not include these, but older versions will had these.

**Pull request merged**